internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -376,6 +377,21 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 
 			"identity": commonschema.SystemOrUserAssignedIdentityOptional(),
 
+			"web_app_routing": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"dns_zone_id": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
```

Comment:
Is it a valid use case/configuration for this to be left blank?

Reply:
Yes, I've confirmed with service team.

Reply:
If it's omitted then the dns_zone_id is set to an empty string "". Where are the DNS entries created for applications that are deployed to the cluster?

Reply:
From service team:
> Probably no dns entries created automatically.

Reply:
Then this should be marked as Required.